### PR TITLE
[16.0][FIX]document_url

### PR DESCRIPTION
**Pull request by DemchukM**
Fixed not working properly when "web_responsive" is installed

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
The new method works. From runboat, I can't test the compatibility with `web_responsive`.

File: document_url/static/src/js/url.esm.js [hunk 1]
```diff
@@ -29,6 +29,18 @@ registerPatch({
     },
 });
 
+registerPatch({
+    name: "Chatter",
+    recordMethods: {
+        /**
+         * Handles click on the attach button.
+         */
```
Comment (chienandalu) [inline document_url/static/src/js/url.esm.js:37]: nitpick:  maybe it'd be more informative to explain *why* we need to handle the click on the attach button.

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Comment by ivs-cetmix** [comment]
Closes https://github.com/OCA/web/issues/2741

**Review by ivs-cetmix — APPROVED** [review]
Functional review LGTM

**Pull request merged**